src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -224,6 +224,20 @@ private async Task PerformV2RestoreAsync(string packagesConfigFilePath, string i
                     ClientPolicyContext = clientPolicyContext
                 };
 
+                IEnumerable<SourceRepository> sourceRepositories = packageRestoreContext.SourceRepositories;
```

Comment:
Don't you have the sources already in line 170ish? 

You are using the context, but the context is actually created just above in the same method.